braze/client.py

```diff
@@ -209,6 +209,96 @@ def identify_users(self, aliases_to_identify):
 
         return self._make_request(payload, BrazeAPIEndpoints.IDENTIFY_USERS, REQUEST_TYPE_POST)
 
+    def create_recipients(self, alias_label, user_id_by_email, trigger_properties_by_email=None):
+        """
+        Create a recipient object using the given the dictionary, `user_id_by_email`
+        containing the user_email key and `lms_user_id` value.
+        Identifies a list of given email addresess with any existing Braze alias records
+        via the provided ``lms_user_id``.
+
+        https://www.braze.com/docs/api/objects_filters/user_alias_object
+        The user_alias objects requires a passed in alias_label.
+
+        https://www.braze.com/docs/api/endpoints/user_data/post_user_identify/
+        The maximum email/user_id dictionary limit is 50, any length beyond 50 will raise an error.
+
+        The trigger properties default to None and return as an empty dictionary if no individualized
+        trigger property is set based on the email.
+
+        Arguments:
+        - `alias_label` (str): The alias label of the user
+        - `user_id_by_email` (dict):  A dictionary where the key is the user's email (str)
+                                    and the value is the `lms_user_id` (int).
+        - `trigger_properties_by_email` (dict) : A dictionary where the key is the user's email (str)
+                                                and the value are the `trigger_properties` (dict)
+                                                Default is None
+
+        Raises:
+        - `BrazeClientError`: if the number of enntries in `user_id_by_email` exceeds 50.
+
+        Returns:
+        - Dict: A dictionary where the key is the `user_email` (str) and the value is the metadata
+                relating to the braze recipient.
+
+        Example: create_recipients(
+                    'alias_label'='Enterprise',
+                    'user_id_by_email'= {
+                        'hamzah@example.com': 123,
+                        'alex@example.com': 231,
+                    },
+                    'trigger_properties_by_email'= {
+                        'hamzah@example.com': {
+                            'foo':'bar'
+                        },
+                        'alex@example.com': {}
+                    },
+                )
+        """
+        if len(user_id_by_email) > GET_EXTERNAL_IDS_CHUNK_SIZE:
```

Comment:
Seems odd to borrow a "chunk size" constant for the purposes of what's effectively a "max size".  Are you sure this is correct, or should we be creating a new `CREATE_RECIPIENTS_MAX_SIZE` variable?

Reply:
Ah, you know what, I led you astrary here, Hamzah.  This constant is associated with a slightly different endpoint, under the `/export/` namespace of the Braze API.
Let's create a new constant `MAX_NUM_IDENTIFY_USERS_ALIASES = 50` and leave a comment next to referring to https://www.braze.com/docs/api/endpoints/user_data/post_user_identify/